packages/sdk/src/swap/__tests__/sdk.test.ts

```diff
@@ -326,6 +326,7 @@ describe(SwapSDK, () => {
           retryDurationBlocks: 500,
           refundAddress: '0xa56A6be23b6Cf39D9448FF6e897C29c41c8fbDFF',
           minPrice: '10000000000000',
+          minPriceX128: '34028236692093846346337460743176821145600000000000000000000000',
```

Comment:
should we send both to the rpc?

Reply:
its not sent to the chainflip rpc, it is the `trpc` handler. i think it should not hurt and it was annoying to remove, so i left it 😅